sklearn/feature_extraction/text.py

```diff
@@ -784,7 +785,8 @@ def _count_vocab(self, raw_documents, fixed_vocab):
 
         analyze = self.build_analyzer()
         j_indices = []
-        indptr = _make_int_array()
+        indptr = []
```

Comment:
I checked that switching from an `array.array` to `list` doesn't have any negative performance impact here. `j_indices` is already a list and that's where most of the data is (since `j_indices` is much longer than `indptr`); `indptr` doesn't really matter. This simplifies things with respect to typing (and possible overflows) though.